indexer/cli/cli.go

```diff
@@ -23,7 +24,11 @@ type Cli struct {
 }
 
 func runIndexer(ctx *cli.Context) error {
-	logger := log.NewLogger(log.ReadCLIConfig(ctx))
+	logger := log.NewLogger(log.CLIConfig{
+		Level:  "warn",
+		Color:  false,
+		Format: "terminal",
+	})
```

Comment:
We need to log if loading the config (which has the logger config in it) fails so provide a logger with normal defaults. Is terminal the correct choice or should I use TEXT?

Reply:
why are we removing `log.ReadCLIConfig`? imo log configuration is the one thing that we should be reading from flags and configuration files

Reply:
Initially was thinking of not having it be flags when i was implementing this but realized log configuration via flags is the most idiomatic way to starting a process

Reply:
Makes sense agree